pydoctor/model.py

```diff
@@ -523,9 +524,14 @@ def setup(self) -> None:
 
 class Attribute(Inheritable):
     kind: Optional[DocumentableKind] = DocumentableKind.ATTRIBUTE
-    annotation: Optional[ast.expr]
+    annotation: Optional[ast.AST]
```

Comment:
I think an annotation should always be an expression. It seems the code that handles the Python 3.9 version of `Subscript` is responsible for this change. I think the mismatch should be handled there instead of by changing the type here and in `node2fullname()` and `node2dottedname()`.

Reply:
Right.